wolfcrypt/src/rsa.c

```diff
@@ -400,6 +422,11 @@ int wc_InitRsaKey_Label(RsaKey* key, const char* label, void* heap, int devId)
             ret = BUFFER_E;
     }
 
+#if defined(HAVE_PKCS11)
+    XMEMSET(key, 0, sizeof(RsaKey));
+    key->isPkcs11 = 1;
```

Comment:
Must do this after `wc_InitRsaKey_ex`

Reply:
Oh. We are setting a member in RsaKey and then calling wc_InitRsaKey_ex(), which will zero the struct. I think that's another reason to make this an argument to wc_InitRsaKey_ex() instead of a member of RsaKey with a very short useful lifetime.

Reply:
It is indeed a short useful lifetime. But during code review, it sounded like this was the preferred solution since adding an argument to `wc_InitRsaKey_ex()` would essentially mean changing the function into a wrapper that would call a new function (that does the same thing `wc_InitRsaKey_ex()` used to do) with this extra argument.